lib/logger.js

```diff
@@ -1,5 +1,5 @@
 import { getLogger } from 'appium-logger';
 
-let logger = getLogger('iosCrashLog');
+let logger = getLogger('iOS');
```

Comment:
hmm, we've been following a convention of having a different log name per package, partially to make it easier to figure out where to grep for log messages if something bad happens.